models/schemas/notificationSchema.ts

```diff
@@ -31,7 +31,7 @@ const NotificationSchema = new mongoose.Schema<Notification>({
   targetModel: {
     type: String,
     required: true,
-    enum: ['User', 'Level'],
+    enum: ['User', 'Level', 'Achievement'],
```

Comment:
target should just be the user (notification source is achievement, and target is the user), so can remove this change